packages/lesswrong/lib/executionEnvironment.ts

```diff
@@ -43,7 +44,13 @@ export const runStartupFunctions = async () => {
 let instanceSettings: any = null;
 export const getInstanceSettings = (): any => {
   if (!instanceSettings) {
-    if (bundleIsServer) {
+    if (isAnyTest) {
+      instanceSettings = {
+        public: {
+          forumType: "EAForum",
```

Comment:
This is actually related to a bigger issue that I've been thinking about for the last couple of days. The reason we need this is that some fields in the schema have different default values depending on `forumTypeSetting`, which means that schema hashes mismatch when we generate them vs. when LessWrong generates them.

I added this code to fix this issue for the unit test which checks generated schemas, but it wasn't until afterwards that I realised this will just cause a more general error every time someone changes a schema.

My current feeling is that we should just completely remove default values from the Postgres schema and let Javascript take care of it (as it does with Mongo). I think this is slightly worse for data integrity, but it won't be any worse than the current situation with Mongo. I can't think of any other solution that isn't just a hopeless mess. It also comes with the added bonus of simplifying the code a bit.

Also tagging @Will-Howard as he has context (and maybe opinions?) here.